_posts/en/newsletters/2021-12-15-newsletter.md

```diff
@@ -0,0 +1,117 @@
+---
+title: 'Bitcoin Optech Newsletter #179'
+permalink: /en/newsletters/2021/12/15/
+name: 2021-12-15-newsletter
+slug: 2021-12-15-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes a proposal to allow relay of
+transactions with zero-value outputs in some cases and summarizes a
+discussion about preparing LN for the adoption of PTLCs.  FIXME:harding
```

Comment:
This seems very high level. As far as I understood the posts on the mailing list the discussion was mainly about preparing the communication protocol for setting up and settling PTLCs with as little roundtrips as possible. Maybe we could write something like: 

> ... summarizes a discussion about preparing LN for a stepwise upgrade to PTLCs.

Reply:
I think this point is covered in the extended description.  I prefer to keep the lede as short as possible.